web/src/components/users/UserEditForm.vue

```diff
@@ -185,6 +186,7 @@ import useSnack from "@/use/use-snack"
 import useUser from "@/use/use-user"
 
 import UserGroupAutocomplete from "@/components/user-groups/UserGroupAutocomplete.vue"
+import useCurrentUser from "@/use/use-current-user"
```

Comment:
Note on import ordering, I put the use imports together, and the component imports last.

My general patter follows https://peps.python.org/pep-0008/#imports, with the secondary consideration of grouping by "relative/conceptual distance". 

i.e. More generic/global-esque things go before less generic/global-esque things.

1. external libraries - aphabetical
2. external components
3. config imports
4. utility imports
5. api imports
6. use file imports
7. local components